lib/bolt/outputter/human.rb

```diff
@@ -136,6 +136,7 @@ def print_task_info(task)
         task_info << "\n\n"
         task_info << "USAGE:\n#{usage}\n\n"
         task_info << "PARAMETERS:\n#{pretty_params}\n" unless pretty_params.empty?
+        task_info << "MODULE:\n#{task['module']}"
```

Comment:
If we don't get this into Puppet 6.0.1, we could derive this from the path of the task's first file.

Reply:
Oh true. I forgot that the first file is guaranteed to be the task file.

Reply:
Well, guaranteed to be *a* task file.

Reply:
Right, but other files in that array can be from other modules, but since the first one is the task itself it'll be from the task module.